### PR TITLE
Utils - Add a simple script to pull out article titles for any Fathead in a new file

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
## Description 
This script takes one argument, the IA id, (e.g. `perl utils/extract_articles.pl angularjs`) and:
- reads its output.txt file line by line
- stores the article titles in an array
- writes the titles to a new "articles.txt" file in the IA dir

## People to notify
@russellholt @moollaza

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by russellholt** [comment]
When writing perl scripts like this, always remember that Larry Wall said he would never have written perl if understood awk.

**Comment by jdorweiler** [comment]
Dax isn't that smart. Everything is an IA as far as he's concerned.

**Comment by jbarrett** [comment]
Why did we back out use of Text::CSV?

File: lib/utils/extract_articles.pl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Read article titles from Output.txt
+open my $fh, '<:encoding(UTF-8)', $file or die "Could not open $file: $!";
+
+my @titles;
+my @fields;
+
+while ( my $line = <$fh>)  {   
+    @fields = split("\t+", $line);
+    $fields[1] =~ /A/ or next;  #skip unless it's an article
+    push @titles, $fields[0]; #first field is the title
```
Comment (jbarrett) [inline lib/utils/extract_articles.pl]: Could probably just write articles.txt here...

File: lib/utils/extract_articles.pl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+close $fh;
+
+# Write list of articles to the new file
+open my $fw, '>:encoding(UTF-8)', $path . "articles.txt" or die();
+
+foreach my $title (@titles) {
```
Comment (jbarrett) [inline lib/utils/extract_articles.pl]: ...unless we wanted to sort `@titles` here - if we're adding these to revision control, it should make the diffing a bit easier.

**Comment by jbarrett** [comment]
@russellholt Please... awk is pure overkill

```bash
grep -P '\tA\t' output.txt | cut -f 1 > articles.txt
```
 :wink:

File: lib/utils/extract_articles.pl (hunk 1 above)
Comment (MariagraziaAlastra) [inline lib/utils/extract_articles.pl]: Lol right

**Comment by MariagraziaAlastra** [comment]
@jbarrett I prefer having less requirements, so since it wasn't super necessary I thought I'd get rid of Text::CSV. Is that ok?

**Comment by jbarrett** [comment]
@MariagraziaAlastra Sure - I think it's fine for this application.

**Pull request merged**